modules/Qnite/plugin/qniteline.cpp

```diff
@@ -47,6 +48,15 @@ QSGNode* QniteLine::updatePaintNode(QSGNode* oldNode, UpdatePaintNodeData*)
   processData();
   int dataSize = xMapped().size();
 
+  // TODO: handle selection highlight in a proper way
```

Comment:
agree. probably we need specific properties to handle the styling of a selected item.